app/controllers/concerns/creates_bulk_actions.rb

```diff
@@ -64,7 +64,8 @@ def validate_job_params(_job_params)
   end
 
   def validate_csv_headers(csv, headers)
-    validator = CsvUploadValidator.new(csv:, headers:)
+    header_validators = headers.present? ? CsvUploadValidator::RequiredHeaderValidator.new(headers:) : []
+    validator = CsvUploadValidator.new(csv:, header_validators:)
```

Comment:
Is the idea here that `CsvUploadValidator`'s collaborators should decide what sort of header validations they want instead of delegating that to the `CsvUploadValidator`?

Reply:
Yes. Previously the only need for validation was to check required headers. However, resolving this issue required additional validation which was only needed by some collaborators.